### ISSUE TITLE
Add California Gothic

### ISSUE THREAD

**Issue by mattlag** [google/fonts#2304]
Hello, Googlefonts!  I just completed work on v2 of my typeface California Gothic.  This version (hopefully) has brought the font up to Google Fonts standards.  I tried to include as many of the recommended items in the repo as possible.  Please let me know if any changes are needed!
https://github.com/mattlag/California-Gothic

You can also check out my site for more background:
https://www.mattlag.com/californiagothic/

Small preview:
![image](https://user-images.githubusercontent.com/5897669/72178448-bc45f100-3397-11ea-8896-255d6e81c97e.png)

## Validation
For the life of me I can't resolve two errors returned by FontBakery.  If you have any ideas on how to fix these I'd be happy to address them and release a v2.1 - or maybe they are okay as-is(?)

###  [code no-license-found]
I literally copied the OFL license URL from the Fontbakery message, and it's still returning an error:
![image](https://user-images.githubusercontent.com/5897669/72178725-5b6ae880-3398-11ea-9b38-8ac39e5c622a.png)
From TTX:
![image](https://user-images.githubusercontent.com/5897669/72178832-953bef00-3398-11ea-8507-f7c654d72fd2.png)

### [code bad-notice-format]
I tried so many different strings here for the copyright notice, and nothing seems to validate.  I finally just left it as a simple string:

![image](https://user-images.githubusercontent.com/5897669/72179085-214e1680-3399-11ea-9c02-971719b44888.png)

Here is the one that (I think) is closest to the recommended format, but it doesn't validate:

![image](https://user-images.githubusercontent.com/5897669/72179640-37100b80-339a-11ea-8b72-3ce2e533b920.png)

**Comment by thlinard** [comment]
Hi @mattlag,

Why don't you take example from a recent commit, like https://github.com/google/fonts/blob/master/ofl/mavenpro/static/ ? Check particularly nameID 0, 13 and 14.

**Comment by mattlag** [comment]
Thank you, @thlinard I was able to fix the `bad-notice-format` error.  I was confused by the Fontbakery message, I thought I was supposed to replace both the `familyname` and the `authors` in the string.

I was also able to fix the NameID 14 License Info error by updating the text in NameID 13 😞 So aparently the `no-license-found` check actually checks NameID 13, but the error message only mentions NameID 14.

I'm going to fix these, and I also noticed one small design error I'd like to fix... so i'll close this and create a 2.1 release, and submit a new issue once I'm done.

Thank you for the help!
